lib/broccoli/ember-app.js

```diff
@@ -139,6 +139,22 @@ EmberApp.prototype._initProject = function(options) {
   @param {Boolean} isProduction
 */
 EmberApp.prototype._initOptions = function(options, isProduction) {
+  var babelOptions = {};
+
+  if (this._addonInstalled('ember-cli-babel')) {
+    var amdNameResolver = require('amd-name-resolver');
+    babelOptions = {
+      babel: {
+        compileModules: true,
+        modules: 'amdStrict',
+        moduleIds: true,
+        exportModuleMetadata: true,
+        sourceMaps: !isProduction,
```

Comment:
This should be based on `options.sourcmaps.enabled` (which is defaulted to `!isProduction` below but can be enabled and disabled independently of a production build).